app/views/user_mailer/assignment_reminder.html.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title></title>
+</head>
+<body>
```

Comment:
Everything above here should go into a layout for emails. See the docs: http://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-layouts